### PR TITLE
Design

### PR DESCRIPTION
**Pull request by luise**

### CODE REVIEW

**Comment by kayousterhout** [comment]
@kklin I can do a first pass at this since I'm familiar with updating all of this stuff

File: source/stylesheets/_variables.scss [hunk 1]
```diff
@@ -59,12 +59,13 @@ $phone-width: $tablet-width - $nav-width !default; // min width before reverting
 // FONTS
 ////////////////////
 %default-font {
-  font-family: "Helvetica Neue", Helvetica, Arial, "Microsoft Yahei","微软雅黑", STXihei, "华文细黑", sans-serif;
-  font-size: 14px;
+  font-family: "OpenSans-Regular", "Helvetica Neue", Helvetica, Arial, "Microsoft Yahei","微软雅黑", STXihei, "华文细黑", sans-serif;
```
Comment (kayousterhout) [inline source/stylesheets/_variables.scss]: Is it intentional to still have Helvetica Neue as the backup here, or should we skip that and go straight to Helvetica or Arial?

File: source/stylesheets/_variables.scss [hunk 2]
```diff
@@ -59,12 +59,13 @@ $phone-width: $tablet-width - $nav-width !default; // min width before reverting
...
+  font-size: 15px;
```
Comment (kayousterhout) [inline source/stylesheets/_variables.scss]: Is it possible to do the font size update in a separate commit? Also I think this is the source of the font-size issues we were seeing earlier: there are a few other places that should be updated I think (which is one reason to do it in a separate commit)

File: source/stylesheets/_variables.scss [hunk 3]
```diff
@@ -23,13 +23,13 @@ under the License.
 
 // BACKGROUND COLORS
 ////////////////////
-$nav-bg: #393939 !default;
+$nav-bg: #b5b4cb !default;
```
Comment (kayousterhout) [inline source/stylesheets/_variables.scss]: are these color changes supposed to be in this commit?

File: source/stylesheets/screen.css.scss [hunk 4]
```diff
@@ -389,7 +401,7 @@ html, body {
   }
 
   p, li, dt, dd {
-    line-height: 1.8;
+    line-height: 1.6;
```
Comment (kayousterhout) [inline source/stylesheets/screen.css.scss]: can this also go in a separate size-changing commit?  You can use this one as a reference: https://github.com/quilt/slate/commit/74e65d362d1af104850ecb8815b01a60142a4b15 (e.g., there's one more place to change the line height -- and it might make sense to just make this a variable)

File: source/stylesheets/_variables.scss [hunk 5]
```diff
@@ -14,6 +14,9 @@ License for the specific language governing permissions and limitations
 under the License.
 */
 
+$light-purple: #CFCFDD;
+$med-purple: #C9C8D9;
+$dark-purple: #7f7f97;
```
Comment (kayousterhout) [inline source/stylesheets/_variables.scss]: super nit but can you format this consistently? (so capitalize the letters, as above)

File: source/stylesheets/screen.css.scss [hunk 6]
```diff
@@ -169,7 +169,6 @@ html, body {
 
   // This is the currently selected ToC entry
   .tocify-focus {
-    box-shadow: 0px 1px 0px $nav-active-shadow;
```
Comment (kayousterhout) [inline source/stylesheets/screen.css.scss]: can you also delete nav-active-shadow in variables.scss (since it looks like now it's not used)?

File: source/stylesheets/_variables.scss (hunk 1 above)
Comment (luise) [inline source/stylesheets/_variables.scss]: I don't really have an opinion. Fine with going straight to Helvetica!

File: source/stylesheets/_variables.scss (hunk 2 above)
Comment (luise) [inline source/stylesheets/_variables.scss]: Yup, I'll do that!

File: source/stylesheets/screen.css.scss (hunk 4 above)
Comment (luise) [inline source/stylesheets/screen.css.scss]: Ah, cool, thanks!

File: source/stylesheets/_variables.scss (hunk 3 above)
Comment (luise) [inline source/stylesheets/_variables.scss]: Ugh, no. Good catch!

**Comment by luise** [comment]
@kayousterhout , thanks for the review! I addressed your comments!
